Develop1

TimurSharibov opened the pull request.

TimurSharibov committed:
Созданы тесты для классов

TimurSharibov committed:
Добавлен отчет jacoco

nikolay-977 commented:
Необходимо исправить: при создании пр-а была указана ветка из репозитория заготовки. Это происходит при создании пр-а по умолчанию. При создании пр-а на это необходимо было обратить внимание(об этом упоминается в инструкции). Чтобы это исправить - необходимо в созданном пул-реквесте нажать кнопку edit и указать ветку main своего репозитория вместо ветки main репозитория заготовки.

TimurSharibov committed:
Добавлен отчет jacoco

TimurSharibov committed:
Add Readme.md

TimurSharibov commented:
Изменено название PR

nikolay-977 commented:
По-прежнему не исправлено
<img width="866" alt="Screenshot 2024-08-10 at 13 13 53" src="https://github.com/user-attachments/assets/34b4343e-3629-4045-957d-cb441fe4101f">

TimurSharibov committed:
1. Антипаттерн магические константы удалены и заменены на переменные отражающие суть.
2. Исправлена кодировка на UTF-8.
3. Добавлены комментарии к тестам.
4. Создан и добавлен новый отчет jacoco.
